src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -778,7 +782,6 @@
     <MergedPayloads Update="@(MergedPayloads)">
       <PayloadGroup>%(FileName)%(Extension)</PayloadGroup>
       <ApkPath>$(MergedPayloadsRootDirectory)%(FileName)%(Extension)/%(FileName)%(Extension).apk</ApkPath>
-      <AppBundlePath>$(MergedPayloadsRootDirectory)%(FileName)%(Extension)/%(FileName)%(Extension).app</AppBundlePath>
```

Comment:
Removing this will break running merged payloads without adding a custom command to the items on line 881.

Reply:
Good point, reverted.